src/backend/joanie/core/tasks.py

```diff
@@ -0,0 +1,20 @@
+"""Celery tasks for the `core` app of Joanie"""
+from logging import getLogger
+
+from django.core.management import call_command
+
+from joanie.celery_app import app
+
+logger = getLogger(__name__)
+
+
+@app.task
+def generate_zip_archive_task(options):
+    """
+    Task to generate the ZIP archive of the signed contracts.
+    It calls the django custom command `generate_zip_archive_of_contracts` with the parsed
+    options.
+    """
+    logger.info("Starting Celery task, generating a ZIP Archive...")
+    call_command("generate_zip_archive_of_contracts", **options)
```

Comment:
Is calling our own command from our own code a good practice ?

Reply:
I don't see real reason to don't do it.
We choose to create a management command to be able to generate a zip from an endpoint or from the commad line.
What do you suggest instead ?

Reply:
To refactor it in a kind of service and then call this service both here and in the management command ?

Reply:
Exactly, yes